.github/workflows/test.yml

```
@@ -295,6 +295,36 @@ jobs:
+          cache-dependency-path: ./wework/package-lock.json
+
+      - name: Install dependencies
+        working-directory: ./wework
+        run: npm ci
+
+      - name: Run wework tests
+        working-directory: ./wework
+        run: npm test -- --coverage --passWithNoTests
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)